doc/userguide/rules/flow-keywords.rst

```diff
@@ -194,9 +194,9 @@ of what such a simple function can do for rule writing. I see a lot of
 applications in things like login tracking, IRC state machines,
 malware tracking, and brute force login detection.
 
-Let's say we're tracking a protocol that normally allows five login
-fails per connection, but we have vulnerability where an attacker can
-continue to login after that five attempts and we need to know about
+Let's say you are tracking a protocol that normally allows five login
+fails per connection, but has a vulnerability where an attacker can
+continue to login after that five attempts and you need to detect
```

Comment:
Since we're here, how about we improve:
```suggestion
continue to login after those five attempts and you need to detect
```

This could go in the typo-fixing commit ;)